cypress/e2e/spec.cy.js

```diff
@@ -4,8 +4,14 @@ describe("empty spec", () => {
   });
 });
 
-describe("visit homepage", () => {
-  it("passes", () => {
+//Routes
+describe("Get started route", () => {
+  it("Visiting the Avatars Page", () => {
     cy.visit("/");
+    cy.get("h1").contains("Alis Learning");
```

Comment:
Great start on the tests 🧪 really good work in getting testing with cypress ready, very clear ✨